### ISSUE TITLE
Doesn't update the pyproject.toml file in source tarball

### ISSUE THREAD

**Issue by mwgamble** [artisanofcode/poetry-stickywheel-plugin#10]
Using this plugin doesn't result in the `pyproject.toml` source tarball being updated. I guess this can be worked around by not publishing the source tarball? Most people will only want and need wheels anyway. It would be good to document this as a limitation though (I'm not sure if updating the `pyproject.toml` file is even feasible).

**Comment by danielknell** [comment]
i mean it is called sticky*wheels* :joy: but when i sort some proper docs ill make sure to note that it wont work with sdists

**Comment by mwgamble** [comment]
Yeah, that did occur to me after I submitted the issue :sweat_smile: I still think it's worth mentioning though.
